source/extensions/filters/http/cache/simple_http_cache/simple_http_cache.cc

```diff
@@ -20,8 +20,10 @@ class SimpleLookupContext : public LookupContext {
   void getHeaders(LookupHeadersCallback&& cb) override {
     auto entry = cache_.lookup(request_);
     body_ = std::move(entry.body_);
+    trailers_ = std::move(entry.trailers_);
     cb(entry.response_headers_ ? request_.makeLookupResult(std::move(entry.response_headers_),
-                                                           std::move(entry.metadata_), body_.size())
+                                                           std::move(entry.metadata_), body_.size(),
+                                                           /*has_trailers=*/trailers_ != nullptr)
```

Comment:
from a stylistic perspective I don't think Envoy uses this `/*has_trailers=*/` convention, though I'm fine with it myself.

Reply:
Removed.